source/ClientCurl.cpp

```diff
@@ -288,7 +288,7 @@ void ClientCurl::HandleInfo::configureCurlHandle() {
 
   curl_easy_setopt(handle, CURLOPT_TIMEOUT, 30);
 
-#if __APPLE__
+#if __APPLE__ | ANDROID
```

Comment:
Change this to `__APPLE__ || ANDROID`

I'd love to change this to 
```
if constexpr __APPLE || ANDROID {}
```
But I assume we have more C++>=11, <17 users than I'd expect.